frontend/app/routes/_public+/apply+/$id+/dental-insurance.tsx

```diff
@@ -54,94 +56,90 @@ export async function action({ request, params }: ActionFunctionArgs) {
+      scrollAndFocusToErrorSummary(errorSummaryId);
+    }
+  }, [actionData]);
+
+  function getErrorMessage(errorI18nKey?: string): string | undefined {
+    if (!errorI18nKey) return undefined;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return t(`dental-insurance.error-message.${errorI18nKey}` as any);
+  }
+
+  const errorMessages = {
+    dentalInsurance: getErrorMessage(actionData?.errors.dentalInsurance?._errors[0]),
+  };
+
+  const errorSummaryItems = createErrorSummaryItems(errorMessages);
 
   const helpMessage = (
     <>
       <ul className="mb-4 list-disc">
-        <li>
-          <Trans ns={handle.i18nNamespaces} i18nKey="apply:dental-insurance-question.list.employer" />
-        </li>
-        <li>
-          <Trans ns={handle.i18nNamespaces} i18nKey="apply:dental-insurance-question.list.pension" />
-        </li>
-        <li>
-          <Trans ns={handle.i18nNamespaces} i18nKey="apply:dental-insurance-question.list.dental-coverage" />
-        </li>
+        <li>{t('dental-insurance.list.employment')}</li>
+        <li>{t('dental-insurance.list.pension')}</li>
+        <li>{t('dental-insurance.list.purchased')}</li>
+        <li>{t('dental-insurance.list.professional')}</li>
         <li className="list-none">
-          <Details id={t('apply:dental-insurance-question.detail.additional-info.title')} title={t('apply:dental-insurance-question.detail.additional-info.title')}>
+          <CollapsibleDetails id={t('dental-insurance.detail.additional-info.title')} summary={t('dental-insurance.detail.additional-info.title')}>
             <div>
-              <p>{t('apply:dental-insurance-question.detail.additional-info.eligible')}</p>
+              <p className="mt-4">{t('dental-insurance.detail.additional-info.not-eligible')}</p>
+              <p className="mt-4">{t('dental-insurance.detail.additional-info.not-eligible-purchased')}</p>
+              <p className="mt-4">{t('dental-insurance.detail.additional-info.eligible')}</p>
               <ul className="mb-4 list-disc pl-6">
-                <li>{t('apply:dental-insurance-question.detail.additional-info.list.opted')}</li>
-                <li>{t('apply:dental-insurance-question.detail.additional-info.list.cannot-opt')}</li>
+                <li>{t('dental-insurance.detail.additional-info.list.opted')}</li>
+                <li>{t('dental-insurance.detail.additional-info.list.cannot-opt')}</li>
               </ul>
-              <Trans ns={handle.i18nNamespaces} i18nKey="apply:dental-insurance-question.detail.additional-info.not-eligible" />
             </div>
-          </Details>
+          </CollapsibleDetails>
         </li>
       </ul>
     </>
   );
 
   return (
-    <Form method="post">
-      {options.length > 0 && (
+    <>
+      {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
+      <Form method="post">
         <div className="my-6">
```

Comment:
This `<div>` should be moved inside of the `options.length > 0` check, otherwise you will render an empty `<div>` with a large margin when `options.length === 0`.